string  module_name   # Name of the module
string  node_name     # Name of the GenICam node
float64 value         # value
---
